Automattic/pocket-casts-ios
# Add Network Connection Testing

Fixes #854 

Adds a status page so users can debug their own network connection. This is mainly a copy of what was done on Android to keep consistency.

| Success | Failure |
| -------- | ------- |
| ![Simulator Screen Shot - iPhone 14 - 2023-05-26 at 17 30 59](https://github.com/Automattic/pocket-casts-ios/assets/7040243/dcc4f4fc-eb53-4e75-965e-808e4401061c) | ![Simulator Screen Shot - iPhone 14 - 2023-05-26 at 17 31 15](https://github.com/Automattic/pocket-casts-ios/assets/7040243/296dd5b3-6663-47c3-b7d8-a0bac6863506) |

Running:

https://github.com/Automattic/pocket-casts-ios/assets/7040243/7f35bbe8-fafa-4fb2-98cb-76f201db78d7

## To test

1. Run the app
2. Go to Profile > Help & Feedback > tap the top right icon
3. Tap "Run now"
4. ✅ All the checks should complete
5. Disable your wi-fi
6. ✅ All tests should fail

## Checklist
